Tests/UnitTestsSerialPort/SerialTests.cs

```diff
@@ -3,7 +3,7 @@
 // See LICENSE file in the project root for full license information.
 //
 
-using nanoFramework.Hardware.Esp32;
+using nanoFramework.Hardware.Esp32; // TODO: only include if platform needs it?!
```

Comment:
please add a comment before that this needs to be commented if tests are run on another platform